### ISSUE TITLE
Standby Shifts

### ISSUE THREAD

**Issue by ToniDS** [c3lingo/c3lingo-stats#1]
If we're planning on having standby shifts again, we should include them, ideally both in the pad-creation script, and in the parsing. Agree?

**Comment by pbock** [comment]
Yes, I suppose having you count them manually isn’t really an ideal workflow.

I’d wait a bit and see what comes of our contemplation of other means of assigning and keeping track of shifts, but if we stick with the pads (fine by me), I agree that we need to include them when generating and parsing the pads.

I’ll keep this open until we know more.
